flink-end-to-end-tests/flink-streaming-kinesis-test/pom.xml

```diff
@@ -151,6 +151,23 @@ under the License.
 					</systemPropertyVariables>
 				</configuration>
 			</plugin>
+
+			<!-- Skip dependency convergence check because of guava version -->
+			<plugin>
```

Comment:
I'm assuming we still want to converge here or at least only exclude Guava.

Could you please check if this legit? @zentol

Reply:
It's a conflict between
```
+-org.apache.flink:flink-streaming-kinesis-test_2.11:1.14-SNAPSHOT
  +-org.apache.flink:flink-connector-kinesis_2.11:1.14-SNAPSHOT
    +-com.google.guava:guava:29.0-jre
```

and
```
+-org.apache.flink:flink-streaming-kinesis-test_2.11:1.14-SNAPSHOT
  +-org.apache.flink:flink-end-to-end-tests-common:1.14-SNAPSHOT
    +-org.apache.flink:flink-connector-testing_2.11:1.14-SNAPSHOT
      +-org.apache.flink:flink-test-utils_2.11:1.14-SNAPSHOT
        +-org.apache.curator:curator-test:2.12.0
          +-com.google.guava:guava:16.0.1
```

Do you have any suggestions for solving this? 😢

Reply:
Okay yes, all good. We are already skipping it in all other e2e tests of connectors.